.mocharc.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
+
+module.exports = {
+  spec: path.join(__dirname, 'packages/**/test/**/**/**/*.spec.js'),
```

Comment:
I created a file in _packages/cli/test/unit/deeply/deeper/nested.spec.js_ and this configuration file picked everything up